src/garage/torch/algos/maml/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Collection of MAML algorithms."""
```

Comment:
i'm not convinced this package needs to exist. can you keep it flat for now? (i.e. put these in .algos)